saphana-plugin/src/main/java/io/cdap/plugin/saphana/SapHanaConstants.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin.saphana;
+
+/**
+ * SAP HANA constants.
+ */
+public class SapHanaConstants {
+
+  public static final String SAPHANA_CONNECTION_STRING_FORMAT = "jdbc:sap://%s:%s/";
+  public static final String PLUGIN_NAME = "SapHana";
+
+  private SapHanaConstants() {
+    throw new AssertionError("Should not instantiate static utility class.");
```

Comment:
just to be consistent with other Constants classes in CDAP, can we remove this constructor?